tests/unit/src/test/scala/tests/codeactions/OrganizeImportsLspSuite.scala

```diff
@@ -59,6 +59,82 @@ class OrganizeImportsLspSuite
     scalacOptions = scalacOption
   )
 
+  check(
+    "basic-unsaved",
+    """
+      |package a
+      |import scala.concurrent.duration._
+      |import scala.concurrent.Future<<>>
+      |import scala.concurrent.ExecutionContext.global
+      |
+      |object A {
+      |  val d = Duration(10, MICROSECONDS)
+      |  val k = Future.successful(1)
+      |}
+      |""".stripMargin,
+    s"${SourceOrganizeImports.title}",
+    """|package a
+       |import scala.concurrent.Future
+       |import scala.concurrent.duration._
+       |// comment
+       |
+       |object A {
+       |  val d = Duration(10, MICROSECONDS)
+       |  val k = Future.successful(1)
+       |}
+       |""".stripMargin,
+    kind = List(sourceKind),
+    scalacOptions = scalacOption,
+    changeFile = (txt: String) => {
+      txt.replace(
+        """|import scala.concurrent.ExecutionContext.global""".stripMargin,
+        """|import scala.concurrent.ExecutionContext.global
+           |import java.nio.file.Path
+           |// comment""".stripMargin
+      )
+    }
+  )
+
+  check(
+    "basic-unsaved-error",
+    """
+      |package a
+      |import scala.concurrent.duration._
+      |import scala.concurrent.Future<<>>
+      |import scala.concurrent.ExecutionContext.global
+      |
+      |object A {
+      |  val d = Duration(10, MICROSECONDS)
+      |  val k = Future.successful(1)
+      |}
+      |""".stripMargin,
+    s"${SourceOrganizeImports.title}",
+    """|package a
+       |import scala.concurrent.duration._
+       |import scala.concurrent.Future
+       |import scala.concurrent.ExecutionContext.global
+       |import java.nio.file.ClassDoNotExist
```

Comment:
It looks like organize wasn't performed. The ordering should be:
```scala
import java.nio.file.ClassDoNotExist

import scala.concurrent.ExecutionContext.global
import scala.concurrent.Future
import scala.concurrent.duration._
```

Reply:
That's right, it's because the file didn't compile and I added a check for that.

Reply:
Oh, yep - found that.